.github/workflows/blank.yml

```diff
@@ -60,17 +62,32 @@ jobs:
             glibc: "2.26"
             linux: "4.14"
             gcc: "7.3.0"
+          - target: aarch64-aws2-linux-gnu
+            vendor: aws2
+            glibc: "2.26"
+            linux: "4.14"
+            gcc: "8.5.0"
           - target: aarch64-aws2023-linux-gnu
             vendor: aws2023
             glibc: "2.34"
             linux: "6.1"
             gcc: "11.3.0"
 
+          - target: x86_64-aws2-linux-gnu
+            vendor: aws2
+            glibc: "2.26"
+            linux: "4.14"
+            gcc: "8.5.0"
           - target: x86_64-aws2-linux-gnu
             vendor: aws2
             glibc: "2.26"
             linux: "4.14"
             gcc: "7.3.0"
+          - target: x86_64-aws2-linux-gnu
+            vendor: aws2
+            glibc: "2.26"
+            linux: "4.14"
+            gcc: "11.3.0"
```

Comment:
seems a duplicate?

Reply:
just want to try upgrading to version 11.3. If it works, I'll check if it can resolve the simdjson error.